Packs/Workday/Integrations/Workday/Workday.py

```diff
@@ -248,7 +260,7 @@ def list_workers(self, page, count, employee_id=None) -> Tuple:
             body = GET_EMPLOYEES_REQ.format(
                 token=self.token, username=self.username, password=self.password, api_version=API_VERSION, page=page,
                 count=count)
-        raw_response = self._http_request(method="POST", url_suffix="", data=body, resp_type='text', timeout=30)
+        raw_response = self._http_request(method="POST", url_suffix="", data=body, resp_type='text', timeout=120)
```

Comment:
is that timeout should really be that long?

Reply:
Unfortunately yes.
it fails with less.